webpack/dev.config.js

```diff
@@ -28,7 +28,7 @@ module.exports = function () {
           include: [resolve('src/'), resolve('test/')],
           options: {
             configFile: resolve('tslint.json'),
-            emitErrors: true,
+            emitErrors: false,
```

Comment:
how come?

Reply:
This was from @Right2Drive's commits. I think it's because the build would fail on linting errors before, this makes them not.